src/GafferImage/Catalogue.cpp

```diff
@@ -1284,7 +1284,14 @@ void Catalogue::compute( ValuePlug *output, const Context *context ) const
 
 		if( boost::starts_with( imageName, g_outputPrefix ) )
 		{
-			outputIndex = std::stoi( imageName.substr( g_outputPrefix.size() ) );
+			try
+			{
+				outputIndex = std::stoi( imageName.substr( g_outputPrefix.size() ) );
+			}	
+			catch( ... )
+			{
+				// If stoi failed, leave the outputIndex at 0
+			}
```

Comment:
Do we need this? Maybe `output:` should be reserved for our use, and people should use a different prefix for their special uses?

Reply:
As mentioned this morning, this fix is just enough to allow Catalogue to output an image saying `Catalogue : Unknown Image "output:foo"` instead of throwing a weird stoi exception - you're probably right that  users shouldn't use the prefix "output:", but this doesn't hurt anyway.